go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/grafana/grafana
 
-go 1.23.1
+go 1.23.2
+
+toolchain go1.23.3
```

Comment:
Do I need these? How can I avoid them, to make merging this PR easier?
Should I downgrade the version of `vitess.io/vitess v0.21.0` from 21?